modules/system/lang/fi/lang.php

```diff
@@ -380,8 +425,8 @@
         'log_theme' => 'Kirjaa teeman muutokset',
         'log_theme_comment' => 'Kun muutos on tehty teemaan käyttäen hallintapaneelia.',
     ],
-    'media' => [
-        'invalid_path' => "Kelvoton tiedostopolku määritetty: ':path'.",
-        'folder_size_items' => 'kohteita',
+        'media' => [
```

Comment:
@mediaclinic please remove tab character.

Reply:
Removed tabs and one comma.